Resolved a few PR comments

satwikgoyal opened the pull request:
Added a home.js for the home component, so that we can now route the pages from App.js. Adding routing through App.js. Added box dimensions for medium and small screen sizes, however, a different UI design should be implemented later. Made a useWindowSize() in gatheredList.js that can later be used to make the entire UI screen size responsive.  Made the menuItems dynamic for later use in notesMenu.js using the .map function.

satwikgoyal committed:
Resolved a few PR comments

Added a home.js for the home component, so that we can now route the pages from App.js. Adding routing through App.js. Added box dimensions for medium and small screen sizes, however a different UI design should be implemented later. Made a useWindowSize() in gatheredList.js that can later be used to make the entire UI screensize responsive.  Made the menuItems dynamic for later use in notesMenu.js using .map function.

rasdy commented on frontend/src/pages/components/gatheredList.js:
```diff
@@ -51,35 +51,57 @@ export default function InfoGatheredList({pages}) {
   return (
     <div className={classes.root}>
-      <Box mt = {6} ml = {8}>
+      <Box mt = {6} ml = {margin_left}>
```
I think you could achieve a similar thing by just using a percentage value, like `ml = "20%"`. If that doesn't have the effect you're looking for, then disregard.

satwikgoyal replied:
Okay, I will do that.

satwikgoyal replied:
Yes, it works, made the change.

rasdy commented on frontend/src/pages/components/notesMenu.js:
```diff
@@ -47,6 +47,12 @@ export default function CustomizedMenus() {
     setAnchorEl(null);
   };
 
+  const gatheredInformation = ["Assignment", "Stakeholder 1", "Stakeholder 2"]
```
What is the CustomizedMenus component used for? It doesn't have to do with the gatheredInfo component does it?

satwikgoyal replied:
I am not sure where we will exactly use it. I think Song and Tim made this file. However, I still made the change to resolve the comment.

satwikgoyal committed:
Updated ml

Changed margin_left to '20%' as they both give similar results. The useWindowSize() function is still there for future use.
